parent/pom.xml

```diff
@@ -1076,6 +1076,8 @@
           <version>${plugin.version.fmt}</version>
           <executions>
             <execution>
+              <id>format</id>
```

Comment:
This id is necessary to be able to refer to the same execution when disabling it on jdk 8 or lower.